lib/modules/filteReddit.js

```diff
@@ -786,18 +801,224 @@ module.go = function() {
 			toggleFilter(subreddit);
 		}
 	);
+};
 
-	if (isPageType('linklist')) {
-		if (module.options.showFilterVisitedLinks.value) {
-			CreateElement.tabMenuItem({
-				text: 'filter visited links',
-			}).checkbox.addEventListener('change', e => {
-				removeVisitedLinksActive = e.target.checked;
-				scanEntries();
-			});
+function getSubredditDefinedFilters() {
```

Comment:
TBH I'm not sure if we need to keep this -- I've never, even seen any subreddit using it. Unless you've seen it used, I'd remove it.